_posts/2011-12-02-on-the-role-of-the-coderetreat-facilitator.md

```diff
@@ -0,0 +1,63 @@
+
+The primary activity of a coderetreat facilitator is to act as a guide both during and between the coding sessions. She should be walking around, observing the approaches pairs are taking, suggesting alternate approaches, constraints, etc. The facilitator should also choose what activities and exercises are appropriate for each session based on the activity of the team.
+
+The more experience a facilitator has, the more alternate methods, the more alternate approaches to the problem she has seen. It is important that the facilitator not code during the sessions and devote their time to facilitation. And facilitating does not include coding with the participants!
+
+You should be visiting each pair and commenting, asking questions, helping them progress.  Having seen these approaches and where they lead, she can help guide others through the self-discovery process.
+
+The first important role of the facilitator is to introduce the day. At a minimum, the introduction should contain five elements:
+
+- a brief history/philosophy of coderetreat;
+- the format of the day;
+- the goals for the day;
+- an explanation of Conway's Game of Life;
+- and, the 4 rules of simple design.
+
+You don't need to provide a great amount of detail around the rules of the game of life, as the first session provides the opportunity for people to dive more deeply into the problem domain. While each of my introductions is slightly different, some focusing more on specifics, some more on the overall goals of the day, the overall message stays consistent. You can get a sense of an introduction from this video that was recorded at the Cleveland coderetreat in January, 2011\. If video does not show up, [click here](http://vimeo.com/18955165).
+
+<object width="400" height="225" classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000" codebase="http://download.macromedia.com/pub/shockwave/cabs/flash/swflash.cab#version=6,0,40,0" _origwidth="400"><param name="allowfullscreen" value="false"><param name="allowscriptaccess" value="never"><param name="wmode" value="opaque"><param name="src" value="http://vimeo.com/moogaloop.swf?clip_id=18955165&amp;server=vimeo.com&amp;show_title=0&amp;show_byline=0&amp;show_portrait=0&amp;color=00adef&amp;fullscreen=1&amp;autoplay=0&amp;loop=0"><embed wmode="opaque" width="400" height="225" type="application/x-shockwave-flash" src="http://vimeo.com/moogaloop.swf?clip_id=18955165&amp;server=vimeo.com&amp;show_title=0&amp;show_byline=0&amp;show_portrait=0&amp;color=00adef&amp;fullscreen=1&amp;autoplay=0&amp;loop=0" allowfullscreen="false" allowscriptaccess="never" _origwidth="400"></object>
```

Comment:
Embedding seems to be broken on this. Have you looked into how we could fix this? Vimeo probably has a JavaScript/HTML5 solution for this I imagine